packages/delisp-core/__tests__/reader.ts

```diff
@@ -116,6 +115,64 @@ describe("Reader", () => {
     });
   });
 
+  it("should read modules", () => {
```

Comment:
Maybe we shouldn't mention modules here. Those tests are at S-Expression level, so we can just say, should be able to read multiple S-expressions.